tests/state_test_util.go

```diff
@@ -148,6 +148,9 @@ func (t *StateTest) Run(subtest StateSubtest, vmconfig vm.Config) (*state.StateD
 		return statedb, fmt.Errorf("post state logs hash mismatch: got %x, want %x", logs, post.Logs)
 	}
 	root, _ := statedb.CommitTo(db, config.IsEIP158(block.Number()))
+	// print state root for evmlab tracing
+	fmt.Printf("{\"stateRoot\": \"%x\"}\n", root)
```

Comment:
This would result in a printout for all tests and whatnot. Imho we should just print `evm` required stuff in `cmd/evm`